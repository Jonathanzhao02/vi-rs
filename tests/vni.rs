mod shared;

fn snapshot_transform(lines: &str) -> String {
    shared::transform_lines(lines, |word| vi::vni::transform_buffer(word.chars()))
}

macro_rules! gen_test_vi {
    ($name:tt, $path: tt) => {
        gen_test!(snapshot_transform, $name, $path);
    }
}
gen_test_vi!(simple_vni, "../testdata/input/simple_vni.txt");
