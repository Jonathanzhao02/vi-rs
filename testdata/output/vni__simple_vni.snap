---
source: tests/vni.rs
expression: snapshot_transform(contents)
info:
  input_file: "../testdata/input/simple_vni.txt"
---
tôi là ai
đây là đâu
thêm một cái nữa
ví tiền
chào
vơ
ru
vươn
chÊ
ă
zzzjjjjhhhkkk
chệch
che8
vươn
a1
hoảng

